CMakeLists.txt

```diff
@@ -110,6 +116,8 @@ option(FLB_AVRO_ENCODER        "Build with Avro encoding support" No)
 option(FLB_AWS_ERROR_REPORTER  "Build with aws error reporting support" No)
 option(FLB_ARROW               "Build with Apache Arrow support"  No)
 option(FLB_WINDOWS_DEFAULTS    "Build with predefined Windows settings"  Yes)
+option(FLB_WASM                "Build with WASM runtime support"  Yes)
```

Comment:
For experimental stuff should we default to off?

Reply:
Yeah, this should be default off but now turned on for CI.
Or, maybe we have to add `-DFLB_WASM=On` task on CI?